.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
+# We don't want to include the tests themselves in the coverage report
+omit = graphql_compiler/tests/*
```

Comment:
Nitpick: can you add a new line at the end of the file, so my editor isn't sad when I open this file?